cea/databases/__init__.py

```diff
@@ -0,0 +1,122 @@
+import os
+from collections import OrderedDict
+
+FILE_EXTENSIONS = ['.xlsx', '.xls', '.csv']
```

Comment:
what about `.dbf` files?

Reply:
This is only for constructing the file structure tree for input databases in `cea.databases`. I used it to get the database categories and the files that are part of that category. e.g. assemblies has ENVELOPE, HVAC and SUPPLY.